styles.css

```diff
@@ -62,12 +161,30 @@ p {
   font-weight: 700;
 }
 
-body {
-  font-family: "Poppins", sans-serif;
-  display: flex;
-  flex-direction: column;
-  height: 100vh;
-  background-color: #f7f7f9;
-  margin: 0;
+.card .head {
+  padding: 0;
+}
+
+.card .text {
   padding: 0;
 }
+
+.card .action {
+  padding: 0;
+}
+
+.card h2 {
+  margin-bottom: 0;
+}
+
+.see-project {
+  color: #396df2;
+  border-radius: 8px;
```

Comment:
- [ ] Please, Add the button styling (Hover, enable, Active, Pressed ) for all your buttons implemented referencing to the Figma template.
Figma buttons styling :
![image](https://user-images.githubusercontent.com/71874425/153681023-88a1eda3-85ac-40b2-b9bb-227feaa059ff.png)